modules/pruning-hard-pruning-registry.adoc

```diff
@@ -112,26 +96,23 @@ To see how many blobs would be removed, run the hard pruner in dry-run mode. No
 changes are actually made:
 +
 ----
-$ oc -n default \
-    exec -i -t "$(oc -n default get pods -l deploymentconfig=docker-registry \
-    -o jsonpath=$'{.items[0].metadata.name}\n')" \
-    -- /usr/bin/dockerregistry -prune=check
+$ oc -n openshift-image-registry \
+    rsh deploy/image-registry \
+    /usr/bin/dockerregistry -prune=check
 ----
 +
 Alternatively, to get the exact paths for the prune candidates, increase the
 logging level:
 +
 ----
-$ oc -n default \
-    exec "$(oc -n default get pods -l deploymentconfig=docker-registry \
-      -o jsonpath=$'{.items[0].metadata.name}\n')" \
-    -- /bin/sh \
-    -c 'REGISTRY_LOG_LEVEL=info /usr/bin/dockerregistry -prune=check'
+$ oc -n openshift-image-registry \
+    rsh deploy/image-registry env REGISTRY_LOG_LEVEL=info \
+    /usr/bin/dockerregistry -prune=check
 ----
 +
 .Truncated sample output
 ----
-$ oc exec docker-registry-3-vhndw \
+$ oc exec openshift-image-registry-3-vhndw \
```

Comment:
the image-registry pod is start with prefix `image-registry`, no openshift in the pod name